assets/app/actions/actionCreators/alertActions.js

```diff
@@ -0,0 +1,39 @@
+import { alertActionTypes as types } from '../../constants';
+
+const authErrorType = types.AUTH_ERROR;
```

Comment:
This makes sense, but if the action type constants are going to be 100% coupled with the action creators, can we just remove the constants file entirely and have the creators be the canonical source of types?

I understand that we need to check that the type of action returned by the action creator matches the string literal of the constant, but it still feels weird to me to expose a property that is just a test hook.....I suppose the alternative is to define the constants again in the test files and that is certainly not what we want.